frontend/app/features/password-reset/components/ResetPasswordForm.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import Button from "@components/button/Button";
+import Input from "@components/input/Input";
+import { useUpdatePasswordMutation } from "@store/api/authSlice";
+import { useRouter, useSearchParams } from "next/navigation";
+import { useState } from "react";
+import styles from "../styles/resetPassword.module.css";
+
+const ResetPasswordPage = () => {
+  const searchParams = useSearchParams();
+  const token = searchParams ? searchParams.get("token") : null;
+  const router = useRouter(); 
+  const [newPassword, setNewPassword] = useState("");
+  const [message, setMessage] = useState("");
+  const [isSuccess, setIsSuccess] = useState(false);
+
+  const [updatePassword, { isLoading }] = useUpdatePasswordMutation();
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    setMessage("");
```

Comment:
the message is initially set to empty string, I assume you need to reset it after this operation done, so if its the case put this inside `finally` state instead
like

```js
try {

} catch () {

} finally {
 
}
```